scene/gui/code_edit.cpp

```diff
@@ -188,13 +189,14 @@ void CodeEdit::_notification(int p_what) {
 					code_completion_line_ofs = CLAMP((code_completion_force_item_center < 0 ? code_completion_current_selected : code_completion_force_item_center) - lines / 2, 0, code_completion_options_count - lines);
 					RenderingServer::get_singleton()->canvas_item_add_rect(ci, Rect2(Point2(code_completion_rect.position.x, code_completion_rect.position.y + (code_completion_current_selected - code_completion_line_ofs) * row_height), Size2(code_completion_rect.size.width, row_height)), theme_cache.code_completion_selected_color);
 
+					const String &lang = TranslationServer::get_singleton()->get_or_add_domain(get_translation_domain())->get_locale();
```

Comment:
`TranslationServer::get_singleton()->get_or_add_domain(get_translation_domain())->get_locale()` could be some helper method in Control, it's verbose and repeated everywhere.